MovieViewer/GoogleService-Info.plist

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
theoretically you should add this to your .gitignore! Someone can now steal your id. :)